src/main/java/com/fundation/logic/view/SearchVideoPanel.java

```diff
@@ -0,0 +1,168 @@
+/**
+ * Copyright (c) 2019 Jalasoft.
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+package com.fundation.logic.view;
+
+import javax.swing.JComboBox;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JTextField;
+import javax.swing.JCheckBox;
+
+/**
+ * Implements the SearchVideoPanel class
+ *
+ * @author John Salazar Pinto
+ * @version 1.0
+ */
+public class SearchVideoPanel extends JPanel {
```

Comment:
This class does many things try to make have a structure and split by class.

Reply:
done